quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -161,10 +161,14 @@ public Map<String, String> getConfigOverrides() {
       return Map.of(
           "polaris.features.defaults.\"ALLOW_SPECIFYING_FILE_IO_IMPL\"",
           "true",
+          "polaris.features.defaults.\"ALLOW_INSECURE_STORAGE_TYPES_ACCEPTING_SECURITY_RISKS\"",
+          "true",
           "polaris.features.defaults.\"INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST\"",
           "true",
           "polaris.features.defaults.\"SUPPORTED_CATALOG_STORAGE_TYPES\"",
-          "[\"FILE\"]");
+          "[\"FILE\",\"S3\"]",
+          "polaris.readiness.ignore-security-issues",
```

Comment:
WDYT about putting this into a global quarkus test config profile?

Reply:
Yea - but I guess all Quarkus profiles should either be special per-test or unified/composable or moved to a central place.

Reply:
composition works by default... one can have `application.propertie` and profile classes at the same time and their properties are merged, right?

Reply:
Yea - but let's defer to a separate PR to clean all those up.